Sources/ProjectDescription/PackageSettings.swift

```diff
@@ -47,25 +47,32 @@ public struct PackageSettings: Codable, Equatable {
     /// Custom project configurations to be used for projects generated from SwiftPackageManager.
     public var projectOptions: [String: Project.Options]
 
+    /// A Boolean value that indicates whether the test targets of local swift packages are included in generated project.
+    public var includeLocalPackageTestTargets: Bool
+
     /// Creates `PackageSettings` instance for custom Swift Package Manager configuration.
     /// - Parameters:
     ///     - productTypes: The custom `Product` types to be used for SPM targets.
     ///     - productDestinations: Custom destinations to be used for SPM products.
-    ///     - baseSettings: Additional settings to be added to targets generated from SwiftPackageManager.
+    ///     - baseSettings: The base settings to be used for targets generated from SwiftPackageManager
     ///     - targetSettings: Additional settings to be added to targets generated from SwiftPackageManager.
     ///     - projectOptions: Custom project configurations to be used for projects generated from SwiftPackageManager.
+    ///     - includeLocalPackageTestTargets: A Boolean value that indicates whether the test targets of
+    ///     local swift packages are included in generated project.
     public init(
         productTypes: [String: Product] = [:],
         productDestinations: [String: Destinations] = [:],
         baseSettings: Settings = .settings(),
         targetSettings: [String: SettingsDictionary] = [:],
-        projectOptions: [String: Project.Options] = [:]
+        projectOptions: [String: Project.Options] = [:],
+        includeLocalPackageTestTargets: Bool = false
```

Comment:
I agree we shouldn't change the default behavior now as it would be breaking, but I do think it should become the default in the future.

Would you mind creating a `BREAKME` file mentioning we want to make the default `true`. `BREAKME` is a new convention introduced by @pepicrft to document what breaking changes we want to do the next time we release a new major version. You can see an example in the [XcodeGraph](https://github.com/tuist/XcodeGraph/blob/main/BREAKME.md) repository.

Reply:
I have saw about BREAKME already! It'll be great chance to introduce BREAKME.